topics/single-cell/tutorials/scatac-batch-correction-snapatac2/workflows/main_workflow.ga

```diff
@@ -0,0 +1,1675 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "This Workflow takes a dataset collection of single-cell ATAC-seq fragments and performs:\n- preprocessing\n- filtering\n- concatenation\n- dimension reduction\n- batch correction\n- leiden clustering",
```

Comment:
Could you also add a Creator (you) and license to the workflow file? (can be done via Galaxy's workflow editor)